spec/game_board_spec.rb

```diff
@@ -32,16 +42,28 @@
   end
 
   describe "Playing spaces with the play_move method" do
-    it "after play_move(:X, 0) spaces[0] = :X" do
-      test_board.play_move(:X, 0)
+    it "after first call to play_move(0) spaces[0] = :X" do
+      test_board.play_move(0)
       expect(test_board.spaces[0]).to eq(:X)
     end
-  end
+     
+    it "call to play_move(1) after play_move(0) set spaces[1] = :O" do
+      test_board.play_move(0)
+      test_board.play_move(1)
+      expect(test_board.spaces[1]).to eq(:O)
+    end
 
+    it "third call to play_move() sets space to :X" do
+      test_board.play_move(2)
+      test_board.play_move(1)
+      test_board.play_move(0)
+      expect(test_board.spaces[0]).to eq(:X)
+    end
+  end 
 
-  describe "clearing board, checking full/empty, number of moves free " do
+describe "clearing board, checking full/empty, number of moves free " do
 
-    it "available_moves returns an array of size 9 when board is empty " do
+  it "available_moves returns an array of size 9 when board is empty " do
```

Comment:
Does it make sense to test the available_moves when the board is not empty? Or is that covered somewhere else?